include/fastrtps/qos/QosPolicies.h

```diff
@@ -616,41 +613,42 @@ class DestinationOrderQosPolicy : public Parameter_t, public QosPolicy
 
 
 /**
- * Class UserDataQosPolicy, to transmit user data during the discovery phase.
+ * Class GenericDataQosPolicy, base class to transmit user data during the discovery phase.
  */
-class UserDataQosPolicy : public Parameter_t, public QosPolicy, public ResourceLimitedVector<rtps::octet>
+class GenericDataQosPolicy : public Parameter_t, public QosPolicy, public ResourceLimitedVector<rtps::octet>
```

Comment:
I'm missing the `push_back` method that is part of the interface on `TopicDataQosPolicy` and `GroupDataQosPolicy`

Reply:
It is part of `ResourceLimitedVector`